Modified docs to adhere to ibm style guide (#57)
devfile/docs: docs/modules/user-guide/partials/proc_adding-a-name-to-a-devfile.adoc

[NOTE]

> If you create workspaces using a factory, defining `name` or `generateName` has the same effect. Use the defined value as the name prefix: `<name>YYYYY` or `<generateName>YYYYY`. If you define both `generateName` and `name`, then the `generateName` takes precedence.

Reviewer: Getting rid of passive in favor of an 'if' clause is not an improvement. Also the second sentence would change the meaning.

```suggestion
Defining `name` or `generateName` has the same effect when creating workspaces using a factory. The value forms the name prefix: `<name>YYYYY` or `<generateName>YYYYY`. When defining both `generateName` and `name`, `generateName` takes precedence.
```
